src/instruments/instrumentstypes.h

```diff
@@ -161,7 +162,14 @@ struct Instrument
     QString abbreviature() const { return !shortNames.isEmpty() ? shortNames.first().name() : QString(); }
 };
 
-using InstrumentList = QList<Instrument>;
+struct PartInstrument {
+    bool part;
```

Comment:
Please, add the word "is" (as well as a default value) to each of bool members here:

bool isPart = false;
bool isSoloist = false;

Reply:
Done.